Islandora-CLAW/islandora_collection
# Remove block installation and include commands.

Resolves: https://github.com/Islandora-CLAW/CLAW/issues/699

Various PRs, comments and tickets:
* https://github.com/Islandora-CLAW/claw_vagrant/pull/48#issuecomment-311535559
* https://github.com/Islandora-CLAW/islandora_collection/pull/22
* https://github.com/Islandora-CLAW/CLAW/issues/670

# What does this Pull Request do?

Removes the default block configuration for the Collection "Members" view. This will now have to be enabled manually by the end-user as it is tied to the theme being used.
The README has been updated with instructions to get back to the same state.

# What's new?
Remove config/install YAML for block, and updated README.

* Does this change require documentation to be updated? yes, included in this PR.
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# How should this be tested?

I guess:
1. Un-install islandora_image and islandora_collection.
1. Pull this branch of islandora_collection.
1. Re-install islandora_collection and islandora_image.
1. Create a collection.
1. Create an image as a member of the collection.
1. Go to the collection, **don't** see the Members block.
1. Follow instructions on README to add Members block.
1. Go to the collection, **do** see the Members block.

# Interested parties
@Islandora-CLAW/committers
